### DISCUSSION TITLE
The current status of this tool?

### DISCUSSION THREAD

**Discussion by aghayoor** [nipreps/dmriprep#173] [Q&A]
Hi developer team,

Based on the roadmap depicted here:
https://www.nipreps.org/dmriprep/master/roadmap.html

It was planned to have v0.9 and the ISMRM tutorial by now. However, I see that the latest release is 0.5.
Could you please clarify about the current stage of this tool? Does it include the whole proposed workflow?

Thank you,
Ali

**Comment by oesteban** [comment]
Hi Ali,

Resources have been limited and we are once again way behind schedule. But the tool is alive and developments are happening. We can reasonably be optimistic that all looks like we'll receive a bump of funding sometime soonish, so that should definitely help.

We also welcome you to the team if you think you can help us push the tool forward with your help :)

Cheers,
Oscar
